usaspending_api/recipient/management/sql/recipient_lookup/080_transactions_parents_missing_name.sql

```diff
@@ -8,7 +8,7 @@ INSERT INTO public.temporary_restock_recipient_lookup (
   uei,
   source,
   parent_duns,
-  parent_legal_business_name
```

Comment:
Just for consistency with step `070` you could get rid of `legal_business_name` on the INSERT and its value, `ultimate_parent_legal_enti` on the SELECT... sine they're all getting set to a NULL `ultimate_parent_legal_enti`